README.md

```diff
@@ -60,35 +58,34 @@ logger = setup_logger(
 )
 
 # Option 1: Load config from MinIO
-cfg_path = "s3a://test-bucket/data-load/config-json/pangenome_ke_genome.json"
+cfg_path = "s3a://cdm-lake/tenant-general-warehouse/kbase/datasets/pangenome_ke-source/config-json/pangenome_ke_genome.json"
 
 # Option 2: Inline JSON config
 cfg_path = r'''
 {
-  "tenant": "pangenome_${dataset}",
-  "dataset": "pangenome_linkml",
+  "tenant": "kbase",
+  "dataset": "ke_pangenome",
   "paths": {
-    "data_plane": "s3a://test-bucket/",
-    "bronze_base": "s3a://test-bucket/pangenome_ke/bronze",
-    "silver_base": "s3a://test-bucket/pangenome_linkml/silver"
+    "data_plane": "s3a://cdm-lake/",
+    "bronze_base": "s3a://cdm-lake/tenant-general-warehouse/kbase/datasets/pangenome_ke-source/bronze/",
   },
   "defaults": {
     "tsv": { "header": true, "delimiter": "\t", "inferSchema": false }
   },
   "tables": [
     {
       "name": "genome",
-      "linkml_schema": "s3a://test-bucket/data-load/linkml-schema/genome_schema.yml",
-      "schema_sql": null,
+      "enabled": true,
+      "schema_sql": "genome_id STRING, gtdb_species_clade_id STRING, gtdb_taxonomy_id STRING, ncbi_biosample_id STRING, fna_file_path_nersc STRING, faa_file_path_nersc STRING",
```

Comment:
for future iterations: allow the user to supply a DDL file

Also: what SQL dialects are supported here? Are Spark's `ArrayType`, `StructType`, `DateType`, `YearMonthIntervalType`, etc., supported? Do you need to specify what is and isn't allowed?